src/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -45,7 +45,7 @@ public T CreateFake<T>(Action<IFakeOptions<T>> optionsBuilder)
         /// </returns>
         public IList<T> CollectionOfFake<T>(int numberOfFakes)
         {
-            var result = new List<T>();
+            var result = new SynchronizedCollection<T>();

```

Comment:
I think it is. FakeItEasy builds the list and returns it, it doesn't keep track of it or make any change to it afterwards. It's the caller's responsibility to use the list in a thread-safe way. To make an analogy, you wouldn't expect `Directory.GetFiles` to return a thread-safe list, and it doesn't.